src/data/VideoFile.py

```diff
@@ -1,10 +1,16 @@
 from abc import ABC, abstractmethod
 import cv2
 import numpy as np
-
+import os
 class VideoStartOrEndOutOfBoundsException(Exception):
     '''Invalid start or end location in VideoFile'''
 
+class VideoNotFoundException(Exception):
```

Comment:
I think these classes should inheret from [Stop Signal](https://github.com/AutoDash/AutoDash/blob/master/src/signals/StopSignal.py) so that the pipeline will stop processing the video if this occurs and move on the the next one

Reply:
actually `RuntimeError` would be better because it prints the error message in that case